orchestrator/core/database/load_seed_data.py

```
@@ -155,27 +154,77 @@ def load_seed_data(load_credentials: bool = True, load_platform_defaults: bool =
+            # + DEFAULT_WORKSPACE_ID (saas ⇒ no-op); idempotent-refresh, never
+            # overwrites edits. Package-qualified imports so the seed shares the
+            # app's ORM session/model objects (module mode: python -m ...).
+            print("\n📂 Loading local-edition first-run content...")
+            try:
+                from core.seeds.seed_local_first_run import seed_local_first_run
+                from core.database.database import get_db_session as _local_session
+
+                with _local_session() as db:
+                    outcome = seed_local_first_run(db)
+                    print(f"  ✅ Local first-run: {outcome}")
+            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)